hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdb/util/ManagedSstFileReader.java

```diff
@@ -125,21 +141,34 @@ public void close() throws UncheckedIOException {
 
   public Stream<String> getKeyStreamWithTombstone(
```

Comment:
This is not used anymore. If so, please remove it.

Reply:
done, removed the constructor